packages/dds/tree/src/test/feature-libraries/editable-tree/editableTree.spec.ts

```diff
@@ -89,19 +90,97 @@ function buildTestPerson(): readonly [SchemaDataAndPolicy, PersonType] {
 }
 
 describe("editable-tree", () => {
-    it("proxified forest", () => {
+    it("can use `Object.keys` and `getOwnPropertyDescriptor` with EditableTree", () => {
         const [, proxy] = buildTestPerson();
-        assert.ok(proxy);
+        assert(isUnwrappedNode(proxy));
+
         assert.equal(Object.keys(proxy).length, 5);
-        assert.deepEqual(proxy[getTypeSymbol](undefined, false), personSchema);
-        assert.deepEqual(proxy.address[getTypeSymbol](undefined, false), addressSchema);
+        assert.deepEqual([...Object.keys(proxy)], ["name", "age", "salary", "friends", "address"]);
```

Comment:
The API does not guarantee the order of keys, so it might be better to test it in an order independent way:
```suggestion
        assert.deepEqual(new Set(Object.keys(proxy)), new Set(["name", "age", "salary", "friends", "address"]));
```

Reply:
If adopting the above, consider adding a check against duplicates in `Object.keys(proxy)`.

Reply:
Done including `Reflect.ownKeys()`.